pkg/controllers/vscr/vdbverify_reconciler.go

```diff
@@ -0,0 +1,69 @@
+/*
+ (c) Copyright [2021-2023] Open Text.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ You may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package vscr
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-logr/logr"
+	v1 "github.com/vertica/vertica-kubernetes/api/v1"
+	vapi "github.com/vertica/vertica-kubernetes/api/v1beta1"
+	"github.com/vertica/vertica-kubernetes/pkg/controllers"
+	verrors "github.com/vertica/vertica-kubernetes/pkg/errors"
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+// VDBVerifyReconciler will verify the VerticaDB in the Vscr CR exists
+type VDBVerifyReconciler struct {
+	VRec *VerticaScrutinizeReconciler
+	Vscr *vapi.VerticaScrutinize
+	Vdb  *v1.VerticaDB
+	Log  logr.Logger
+}
+
+func MakeVDBVerifyReconciler(r *VerticaScrutinizeReconciler, vscr *vapi.VerticaScrutinize,
+	log logr.Logger) controllers.ReconcileActor {
+	return &VDBVerifyReconciler{VRec: r, Vscr: vscr, Log: log, Vdb: &v1.VerticaDB{}}
```

Comment:
Can we include the reconciler in the log name?
```suggestion
	return &VDBVerifyReconciler{VRec: r, Vscr: vscr, Log: log.WithName("VDBVerifyReconciler"), Vdb: &v1.VerticaDB{}}
```